CI/scripts/build_bsp.sh

```diff
@@ -104,6 +104,9 @@ cp -r scripts/boot/* ../hdl/vendor/AnalogDevices/vivado/projects/common/boot/
 
 # Copy additional IP files
 cp -r scripts/library/* ../hdl/vendor/AnalogDevices/vivado/library/
+sed -i 's@source ../../scripts/adi_env.tcl@source ../../../scripts/adi_env.tcl@' $TARGET/library/util_sync/sync_delay/util_delay_ip.tcl
+sed -i 's@source ../../scripts/adi_env.tcl@source ../../../scripts/adi_env.tcl@' $TARGET/library/util_sync/sync_fast_to_slow/sync_fast_to_slow_ip.tcl
+sed -i 's@source ../../scripts/adi_env.tcl@source ../../../scripts/adi_env.tcl@' $TARGET/library/util_sync/sync_slow_to_fast/sync_slow_to_fast_ip.tcl
```

Comment:
This should be done in the source files themselves since it will make them easier to upstream.

Reply:
Fixed.